test/Datadog.Trace.Tests/ExporterWriterBufferTests.cs

```diff
@@ -35,7 +35,7 @@ public void Push_MoreThanCapacity_False()
 
             Assert.False(buffer.Push(101));
 
-            // Check that one random element of the queue was replaced
+            // Check NONE element of the buffer was replaced
```

Comment:
We should leave this file unchanged as well since it gets removed the Datadog trace buffer PR